app/controllers/foreman_bootdisk/api/v2/disks_controller.rb

```diff
@@ -10,7 +10,6 @@ class DisksController < ::Api::V2::BaseController
           api_base_url "/bootdisk/api"
         end
 
-        before_filter :find_resource, :only => :host
```

Comment:
To summarise, you want to move the subnet action from v2/disks_controller into new controller like v2/subnet_disks_controller?

I don't follow what this change has anything to do with "pikcing up fixes and features from Foreman"